### PR TITLE
Gio icons

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**

https://github.com/gravitee-io/issues/issues/6894

**Description**

Add gravitee icons set 

**Additional context**

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-imwffbcorv.chromatic.com)

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.0.1-gio-icons-23b0a72
# or
yarn add @gravitee/ui-particles-angular@2.0.1-gio-icons-23b0a72
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.0.1-gio-icons-23b0a72
# or
yarn add @gravitee/ui-policy-studio-angular@2.0.1-gio-icons-23b0a72
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.0.1-gio-icons-1549a5a
# or
yarn add @gravitee/ui-policy-studio-angular@2.0.1-gio-icons-1549a5a
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.0.1-gio-icons-1549a5a
# or
yarn add @gravitee/ui-particles-angular@2.0.1-gio-icons-1549a5a
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.0.1-gio-icons-eb03d18
# or
yarn add @gravitee/ui-particles-angular@2.0.1-gio-icons-eb03d18
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.0.1-gio-icons-eb03d18
# or
yarn add @gravitee/ui-policy-studio-angular@2.0.1-gio-icons-eb03d18
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.0.1-gio-icons-293d56d
# or
yarn add @gravitee/ui-policy-studio-angular@2.0.1-gio-icons-293d56d
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.0.1-gio-icons-6395a94
# or
yarn add @gravitee/ui-policy-studio-angular@2.0.1-gio-icons-6395a94
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.0.1-gio-icons-6395a94
# or
yarn add @gravitee/ui-particles-angular@2.0.1-gio-icons-6395a94
```

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-icons/sb-get-icons-list.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import { DomSanitizer } from '@angular/platform-browser';
+import { isEmpty } from 'lodash';
+
+@Pipe({ name: 'sbGetIconsList' })
+export class SbGetIconsListPipe implements PipeTransform {
+  private icons: string[] = [];
+
+  constructor(matIconRegistry: MatIconRegistry, sanitizer: DomSanitizer) {
+    matIconRegistry.getSvgIconFromUrl(sanitizer.bypassSecurityTrustResourceUrl('assets/gio-icons.svg')).subscribe(icon => {
+      icon.querySelectorAll('symbol').forEach(icon => this.icons.push(icon.id));
```
Comment (jgiovaresco) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-icons/sb-get-icons-list.pipe.ts:27]:
```suggestion
      this.icons = icon.querySelectorAll('symbol').map(icon => icon.id);
```

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-icons/sb-get-icons-list.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    });
+  }
+
+  public transform(_: never, search: string): string[] {
+    console.log(search);
```
Comment (jgiovaresco) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-icons/sb-get-icons-list.pipe.ts:32]: I think we can remove this

**Review by jgiovaresco — APPROVED** [review]

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-icons/sb-get-icons-list.pipe.ts (hunk 1 above)
Comment (ThibaudAV) [reply ui-particles-angular/projects/ui-particles-angular/src/lib/gio-icons/sb-get-icons-list.pipe.ts:27]: nope, querySelectorAll dose not return array but iterable things

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.0.2-gio-icons-6da0d6f
# or
yarn add @gravitee/ui-particles-angular@2.0.2-gio-icons-6da0d6f
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.0.2-gio-icons-6da0d6f
# or
yarn add @gravitee/ui-policy-studio-angular@2.0.2-gio-icons-6da0d6f
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.0.2-gio-icons-c8e00f7
# or
yarn add @gravitee/ui-policy-studio-angular@2.0.2-gio-icons-c8e00f7
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.0.2-gio-icons-c8e00f7
# or
yarn add @gravitee/ui-particles-angular@2.0.2-gio-icons-c8e00f7
```

**Review by ytvnr — APPROVED** [review]

**Pull request merged**